passes/opt/opt_lut.cc

```diff
@@ -147,23 +147,23 @@ struct OptLutWorker
 					{
 						if (lut_width <= dlogic_conn.first)
 						{
-							log("  LUT has illegal connection to %s cell %s.%s.\n", lut_dlogic->type.c_str(), log_id(module), log_id(lut_dlogic));
-							log("    LUT input A[%d] not present.\n", dlogic_conn.first);
+							log_debug("  LUT has illegal connection to %s cell %s.%s.\n", lut_dlogic->type.c_str(), log_id(module), log_id(lut_dlogic));
```

Comment:
Should something as serious sounding as an "illegal connection" be promoted to be a warning message?

Reply:
That just means it doesn't know which `SB_LUT4` is the `SB_CARRY` connected to. Ideally this entire chunk of logic would be removed in favor of `SB_CARRY_LUT4` cell; it's a hack.